### PR TITLE
feat: updates for openai plugins

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

> **Note**
> This PR is based on the branch `v1.3.1`.

- add config `alwaysIssueRefreshToken` for web apps to unblock OAuth integrations that are not strictly conform OpenID Connect. when it's enabled, Refresh Tokens will be always issued regardless if `prompt=consent` was present in the authorization request
- parse requests with `application/json` content-type for `/oidc` APIs to increase compatibility
- merge #3873 

## Testing

integration tests

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [x] integration tests
- [ ] docs (will write some blog posts)

### CODE REVIEW

File: packages/phrases/src/locales/de/translation/admin-console/application-details.ts [hunk 1]
```diff
@@ -41,6 +41,9 @@ const application_details = {
   enable_admin_access: 'Admin-Zugang aktivieren',
   enable_admin_access_label:
     'Zugang zur Management API aktivieren oder deaktivieren. Wenn aktiviert, können Access Tokens verwendet werden, um die Management API im Namen der Anwendung aufzurufen.',
+  always_issue_refresh_token: 'Always issue Refresh Token',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/de/translation/admin-console/application-details.ts:44]: nit: why is this line not translated. lol
```suggestion
  always_issue_refresh_token: 'Geben Sie immer ein Aktualisierungstoken aus',
```

File: packages/phrases/src/locales/fr/translation/admin-console/application-details.ts [hunk 2]
```diff
@@ -41,6 +41,9 @@ const application_details = {
   enable_admin_access: "Activer l'accès administrateur",
   enable_admin_access_label:
     "Activer ou désactiver l'accès à l'API de gestion. Une fois activé, vous pouvez utiliser des jetons d'accès pour appeler l'API de gestion au nom de cette application.",
+  always_issue_refresh_token: 'Always issue Refresh Token',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/fr/translation/admin-console/application-details.ts:44]:
```suggestion
  always_issue_refresh_token: 'Émettez toujours un jeton d'actualisation',
```

Comment (gao-sun) [reply packages/phrases/src/locales/fr/translation/admin-console/application-details.ts:44]: thanks this is awkward

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by darcyYe — APPROVED** [review]

**Pull request merged**